fix(docs): Clarify why removing .wslconfig in the UP4W tutorial (#1309)
canonical/ubuntu-pro-for-wsl: docs/tutorials/getting-started-with-up4w.md

unless a distro with the name "Ubuntu" is already installed.
```

> Certain WSL global configurations may block the communication between the Ubuntu

Reviewer: ```suggestion
Global configuration settings for WSL may block the communication between the Ubuntu
```

> Pro for WSL and the WSL instances. For simplicity, if you already have WSL

Reviewer: ```suggestion
Pro for WSL application and instances of Ubuntu on WSL. If you already have WSL
```

> installed, with `~\.wslconfig` on your system, you are advised to backup the

Reviewer: ```suggestion
installed, with `~\.wslconfig` on your system, we recommend that you back up the
```

> file then remove it before continuing the tutorial.

Reviewer: ```suggestion
file, then remove it, before continuing the tutorial.
```

> For more information about `.wslconfig`, refer to [WSL docs](https://learn.microsoft.com/en-us/windows/wsl/wsl-config#wslconfig)

Reviewer: ```suggestion
For more information about `.wslconfig`, refer to the [WSL documentation from Microsoft](https://learn.microsoft.com/en-us/windows/wsl/wsl-config#wslconfig).
```
